source/cloud/ibm/iks.md

```diff
@@ -0,0 +1,76 @@
+# IBM Kubernetes Service (IKS)
+
+RAPIDS can be deployed on IBM Cloud via IBM Cloud managed Kubernetes service (IKS) using any of the [supported Kubernetes installation methods](../../platforms/kubernetes).
+
+## Install pre-requisites
+
+Install and configure dependencies in your local environment: [kubectl](https://kubernetes.io/docs/tasks/tools/), [helm](https://helm.sh/), [IBM cloud cli](https://cloud.ibm.com/docs/cli?topic=cli-getting-started) and [IBM Kubernetes Service (KS) plugin](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install).
+
+## Login to IBM CLI
+
+```shell
+$ ibmcloud login -a cloud.ibm.com -r <region>
+$ ibmcloud target -g <resource group>
+```
+
+```{note}
+You can list regions with `$ ibmcloud regions` and resource groups with `$ ibmcloud resource groups`.
+```
+
+## Create a Kubernetes cluster
+
+```shell
+$ ibmcloud ks cluster create classic \
+    --name <CLUSTER_NAME> \
+    --zone dal10 \
+    --flavor gx2-8x64x1v100 \
```

Comment:
I am unable to find any `gx2` flavor instances, are these still available? Do we need to enable something specific on our account.

cc @marifse 

```console
$ ibmcloud ks flavors --zone dal10
OK
For more information about these flavors, see 'https://ibm.biz/flavors'
Name                      Cores   Memory   Network Speed   OS             Server Type   Storage   Secondary Storage   Flavor Class   Provider
b3c.16x64                 16      64GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
b3c.16x64.300gb           16      64GB     1000Mbps        UBUNTU_18_64   virtual       25GB      300GB               -              classic
b3c.32x128                32      128GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
b3c.4x16                  4       16GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
b3c.56x242                56      242GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
b3c.8x32                  8       32GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
c3c.16x16                 16      16GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
c3c.16x32                 16      32GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
c3c.32x32                 32      32GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
c3c.32x64                 32      64GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.16x128                16      128GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.30x240                30      240GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.48x384                48      384GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.4x32                  4       32GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.56x448                56      448GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.64x512                64      512GB    1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
m3c.8x64                  8       64GB     1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
mb4c.20x192               20      192GB    10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mb4c.20x384               20      384GB    10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mb4c.20x64                20      64GB     10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mb4c.20x64.2x1.9tb.ssd    20      64GB     10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mb4c.32x384.3.8tb.ssd     32      384GB    10000Mbps       UBUNTU_18_64   physical      2TB       1.92TB              -              classic
mb4c.32x384.6x3.8tb.ssd   32      384GB    10000Mbps       UBUNTU_18_64   physical      2TB       1.92TB              -              classic
mb4c.32x768.3.8tb.ssd     32      768GB    10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mb4c.48x1536              48      1536GB   10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
me4c.4x32                 4       32GB     10000Mbps       UBUNTU_18_64   physical      2TB       2TB                 -              classic
me4c.4x32.1.9tb.ssd       4       32GB     10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mg4c.32x384.2xp100        32      384GB    10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
mg4c.48x384.2xv100        48      384GB    10000Mbps       UBUNTU_18_64   physical      2TB       960GB               -              classic
u3c.2x4†                  2       4GB      1000Mbps        UBUNTU_18_64   virtual       25GB      100GB               -              classic
```

Reply:
hey Jacob! thank you four response. To provision IBM Kubernetes cluster, you can use the GPU enabled mg4c.32x384.2xp100 (bare-metal server) instance, though it will incur charges per month one time not hourly, so would be careful. And for your information the  gx2-8x64x1v100 is only available for Virtual Server Instance, which is a virtual instance and charged per hour basis. Thank you

Reply:
Thanks for the response @marifse. 

I'm apprehensive about providing instructions on deploying a very expensive cluster that bills monthly 😅. Your original instructions used `gx2-8x64x1v100` which I've just copied over here, is there any way to get per-hour billing for GPUs on IKS?

We also need to test these instructions for every RAPIDS release (every two months) and right now this means launching two nodes and checking that everything runs. We can't really justify ~$1k per test because of a limitation around the billing period.

Reply:
@marifse gentle nudge here, do you have any suggestions on how we can safely document IKS?

Reply:
@jacobtomlinson sorry for being late, i have communicated to IBM guy, for now there is no option in IKS (IBM Kubernetes Services) to use GPU enabled instance without bare-metal profile.
For testing  the instructions for every RAPIDS release, i have communicated to IBM guy, he will sort out some solution for this, as soon as he will be back to work, as he is on leave for next 15 days.

Reply:
Thanks @marifse I'll leave this as a draft in the meantime, but the single-node docs are already merged and will be in the next stable release which we hope to get out this week.